GOVERNANCE.md

```diff
@@ -0,0 +1,416 @@
+an us vs. them issue but as a collaborative »we have to solve this together.«
+
+Using votes to decide on forks could be seen as detrimental to this goal, since
+a vote has loser by design—if you are not voting for me, you're voting against
+me.
+
+
+### Exit costs
+
+One of our goal is to make exit costs low, i.e. allow users to easily abandon
+undesirable forks. But it can be argued that if people have very low exit costs
+for a given system, then their incentive to vote according to their true beliefs,
+which should be the main goal for any sort of vote, is lowered and in turn would
+then render the vote pointless.
+
+And while the exit costs for a single voter can be quite low, the costs a fork
+imposes on a community can still end up very high, especially when it results in
+two persistent chains. Both forks will have a vested interest in keeping the
+»original« name, code will have to be adjusted, wallets to be updated, nodes
+changed and users will definitely end up being confused.
+
+
+### Constitution
+
+In our context a constitution could provide the most important invariants that
+should be upheld both in the technical and human domain, e.g. across protocol
+changes the total supply must never exceed n coins or the Aeternity Foundation
+shall produce detailed reports of its allocation of funds.
+
+Thus a constitution specifies what is allowed to be decided on. It should be
+noted however that these rules on how to change rules are informed by expectations
+of what the outcomes should or could be., which could lead to problems if these
+rules were to be enforced by computers, if it is even possible to encode all of
+them as »smart contracts«.
+
+
+### Automatic protocol amendments
+
+Automatic protocol amendments are a mechanism where updates to the node software
+run by participants are tracked inside the system and in most cases adoption
+is also being voted upon.
+The most obvious issue with this approach is that for this system to work for
+all clients, the changes need to be expressed in a way that can be understood by
+everyone, which means that everyone has to either run the same software or there
+needs to be a common language. This could be achieved by either encoding all the
+relevant consensus rules in a language for a virtual machine every participant
+has to run or by using a language for these change that can be transpiled easily.
+
+Now if we assume that we have such a language, then the next step is to verify
+that the given piece of code adheres to the constitution and does not violate
+any of its invariants. This could either be done via manual, human review or
+formal methods, i.e. having both the constitution and the code be formally
+verifiable. The problem is, that this is most likely a gross oversimplification
+because software engineering is never as simple as just writing a single piece
+of code and have it just magically work, especially not
```

Comment:
This sentence is confusing, it's seams to and abruptly. At least it's missing a `.` at the end.